__tests__/utility.service.test.js

```diff
@@ -19,18 +20,51 @@ describe("UtilityService", () => {
   });
 
   describe("updateSearchResults", () => {
-    it("should call provided functions", () => {
+    it("should call provided functions and return expected objects", () => {
```

Comment:
Non-blocking: can this be more descriptive? This test name is applicable to pretty much all the unit tests